modules/persistent-storage-csi-snapshots-delete.adoc

```diff
@@ -24,6 +21,67 @@ metadata:
 driver: hostpath.csi.k8s.io
 deletionPolicy: Delete <1>
 ----
-<1> If the `Delete` value is set, the underlying snapshot will be deleted, along with the `VolumeSnapshotContent` object. If the `Retain` value is set, both the underlying snapshot and `VolumeSnapshotContent` object remain.
+<1> When deleting the volume snapshot, if the `Delete` value is set, the underlying snapshot will be deleted, along with the `VolumeSnapshotContent` object. If the `Retain` value is set, both the underlying snapshot and `VolumeSnapshotContent` object remain.
   +
-If the `Retain` value is set, and the `VolumeSnapshot` object is deleted without deleting the corresponding `VolumeSnapshotContent` object, then the content will remain. The snapshot itself is also retained in the storage back end.
+If the `Retain` value is set and the `VolumeSnapshot` object is deleted without deleting the corresponding `VolumeSnapshotContent` object, the content will remain. The snapshot itself is also retained in the storage back end.
+
+. Delete the volume snapshot:
+
+.. If the deletion policy is set to `Delete`, enter the following command:
```

Comment:
I think we should combine the deletion of the snapshot into a single step. Then we can just include the steps to delete the VolumeSnapshotContent in the one-off case of the deletion policy being `Retain` and the user wanting to delete this content later.

Reply:
Resolved.